docs/deploy-from-precompiled-binaries-windows.md

```diff
@@ -0,0 +1,105 @@
+---
+date: '2019-10-16'
+title: 'Deploy Platform V3 from precompiled binaries on Windows'
+layout: docs
+---
+## Summary
+
+Use this guide to <a class="crosslink" href="https://virtocommerce.com/ecommerce-hosting" target="_blank">deploy</a> and configure precompiled Virto Commerce Platform V3.
+
+### Prerequisites
+
+[Prerequisites for .NET Core on Windows](https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites)
+
+[.NET Core SDK](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-windows-x64-installer)
```

Comment:
1. why SDK, isn't Runtime enough?
2. it's .NET 2.2 app. Why to point to specific ver. 3.0.100? Will it work?

Reply:
1. As i understend Runtime does not contain `dotnet` tool
2. Link to latest dotnet core SDK it works :)